spec/model_decorator_spec.rb

```diff
@@ -100,7 +111,7 @@
       let(:partitions) { [] }
 
       it "calls data_source_exists? with table name" do
```

Comment:
Tests present for data_source_exists? Vs table_exists?

Reply:
Line 14 is used as a dynamic expectation based on the Rails version. But you're right, this test should be renamed. Also, I don't think this is tested at the integration level. I'll add some. Thanks!